bundle/config/mutator/paths/dashboard_paths_visitor.go

```diff
@@ -0,0 +1,18 @@
+package paths
+
+import (
+	"github.com/databricks/cli/libs/dyn"
+)
+
+func VisitDashboardPaths(value dyn.Value, fn VisitFunc) (dyn.Value, error) {
+	pattern := dyn.NewPattern(
+		dyn.Key("resources"),
+		dyn.Key("dashboards"),
+		dyn.AnyKey(),
+		dyn.Key("file_path"),
+	)
+
+	return dyn.MapByPattern(value, pattern, func(path dyn.Path, value dyn.Value) (dyn.Value, error) {
+		return fn(path, TranslateModeLocalAbsoluteFile, value)
```

Comment:
@anton-107 FYI, to inline the dashboard contents instead of passing the file path to Terraform (to make it work in the workspace), this will need to be a local relative path. That's needed to be able to use `b.SyncRoot` to read its contents.